src/server/frontend_wayland/wayland_connector.cpp

```diff
@@ -759,7 +760,10 @@ mf::WaylandConnector::WaylandConnector(
 
     // The XDG shell support is currently too flaky to enable by default
     if (getenv("MIR_EXPERIMENTAL_XDG_SHELL"))
-        xdg_shell_global = std::make_unique<XdgShellV6>(display.get(), shell, *seat_global);
+    {
+        xdg_shell_unstable_v6_global = std::make_unique<XdgShellUnstableV6>(display.get(), shell, *seat_global);
+        xdg_shell_stable_global = std::make_unique<XdgShellStable>(display.get(), shell, *seat_global);
+    }
```

Comment:
I think these should be independent:
 - we probably want to enable unstable soon; but,
 - stable should probably be off by default until we've validated the behavior against at least one toolkit.

MIR_EXPERIMENTAL_XDG_SHELL_STABLE?

Reply:
done